### ISSUE TITLE
FlareSolverr

### ISSUE THREAD

**Issue by blanksigns** [pijarr/pijarr#26]
Hi there,

Can we please add FlareSolverr?

it works well with Prowlerr and allows us to handle indexers using cloudflare 

Thanks,

**Comment by Maxsafer** [comment]
its very easy to install inside a container as well

**Comment by jamiebarry148-sketch** [comment]
> Hi there,
> 
> Can we please add FlareSolverr?
> 
> it works well with Prowlerr and allows us to handle indexers using cloudflare
> 
> Thanks,

Agreed... can flaresolverr be added as well as media servers players 😇
